injector/src/test/java/edu/ucr/cs/riple/injector/OnLocalVariableInjectionTest.java

```diff
@@ -244,9 +244,9 @@ public void onArrayType() {
             "import edu.ucr.UnTainted;",
             "public class Foo<T> {",
             "   public void foo() {",
-            "      @UnTainted Map<String, T[]>[] f0;",
```

Comment:
This failure is a bug in your changes not that the expected output is wrong.

Reply:
Reverted to original